metisp/pymetis/src/pymetis/recipes/lm_img/metis_lm_img_basic_reduce.py

```diff
@@ -93,22 +93,6 @@ class ProductBasicReduced(TargetSpecificProduct):
         def tag(cls) -> str:
             return rf"LM_{cls.target():s}_BASIC_REDUCED"
 
-
-
-    def prepare_flat(self, flat: cpl.core.Image, bias: cpl.core.Image | None):
-        """ Flat field preparation: subtract bias and normalize it to median 1 """
-        Msg.info(self.__class__.__qualname__, "Preparing flat field")
-
-        if flat is None:
-            raise RuntimeError("No flat frames found in the frameset.")
-        else:
-            if bias is not None:
-                flat.subtract(bias)
-            median = flat.get_median()
-            return flat
-
-            # return flat.divide_scalar(median)
-
     def prepare_images(self,
```

Comment:
This function doesn't seem to do much anymore, except load the raw images. At least the method parameters `bias` and `flat` can be removed as they are not used.